lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -1,7 +1,7 @@
 # typed: strict
 # frozen_string_literal: true
 
-RubyLsp::Addon.depend_on_ruby_lsp!(">= 0.19.1", "< 0.20")
+RubyLsp::Addon.depend_on_ruby_lsp!(">= 0.20", "< 0.21")
```

Comment:
Is it possible to change it locally instead during development? I'm afraid of it degrading the internal testing since Ruby LSP isn't automatically bumped right away in my experience.